src/python/pants/core/util_rules/environments.py

```diff
@@ -190,6 +190,16 @@ class DockerImageField(StringField):
         The choice of image ID can affect the reproducibility of builds. Consider using an
         immutable digest if reproducibility is needed, but regularly ensure that the image
         is free of relevant bugs or security vulnerabilities.
+
+        Note that in order to use an image as a `docker_environment` it must have a few tools:
+        - `/bin/sh`
+        - `/usr/bin/env`
+        - `bash`
+        - `tar`
+
+        While most images will have these preinstalled, users of base images such as Distroless
```

Comment:
Due to an extremely annoying linebreaking "feature" of readme.com, we'll need the entire paragraph current spread across lines 200-202 to be on a single long line.

We're hoping to move to another docs hosting solution soon, but until then, alas, we have to work with readme's constraints.

Reply:
@benjyw fyi, the `help_text()` helper does a `softwrap` here, so this wasn't needed ;) (see the lines above, also broken into sensible line lengths)

Reply:
Ah right, my bad, that problem is just in raw `.md` files, not in generated docs. Brain fart on my part.